readme.md

```diff
@@ -1,21 +1,21 @@
-# Colored blocks
-
-Display three colored blocks on a page one below another. Each block 
-must be 100 pixels high and span the entire width of the page.
-The top block must be red, the middle one must be green, and the bottom one
-must be blue.
-
-Your browser will add a narrow margin to the `<body>` element by default. Remove 
-it so that there is no space between the edges of the page and the blocks.
-
-You can find some sample screenshots in the [backstop_data/bitmaps_reference](backstop_data/bitmaps_reference) folder.
-
----
-**Read the guideline before start**
-
-[Guideline](https://github.com/mate-academy/layout_task-guideline/blob/master/README.md)
-
-**Result**
-
-[UPDATE DEMO LINK](https://<your_account>.github.io/<repo_name>/) <br>
-[UPDATE TEST REPORT LINK](https://<your_account>.github.io/<repo_name>/report/html_report/)
+# Colored blocks
+
+Display three colored blocks on a page one below another. Each block 
+must be 100 pixels high and span the entire width of the page.
+The top block must be red, the middle one must be green, and the bottom one
+must be blue.
+
+Your browser will add a narrow margin to the `<body>` element by default. Remove 
+it so that there is no space between the edges of the page and the blocks.
+
+You can find some sample screenshots in the [backstop_data/bitmaps_reference](backstop_data/bitmaps_reference) folder.
+
+---
+**Read the guideline before start**
+
+[Guideline](https://github.com/mate-academy/layout_task-guideline/blob/master/README.md)
+
+**Result**
+
+[UPDATE DEMO LINK](https://<your_account>.github.io/<repo_name>/) <br>

```

Comment:
You forgot about links.
`PR must include updated readme.md with Demo and Test report`